<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeightMapRadiusRatio</class>
 <widget class="QDialog" name="HeightMapRadiusRatio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok_btn_action">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn_action">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_box_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Smooth Step:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="smooth_step_action_"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>50</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ratios : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ratio_action_"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height Multiplier :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="multiplier_action_"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
